tests/common.go

```diff
@@ -3844,3 +3845,109 @@ func IsCloudDriveInitialised(n node.Node) (bool, error) {
 	}
 	return false, err
 }
+
+//WaitForExpansionToStart waits for pool expansion to trigger
+func WaitForExpansionToStart(poolID string) error {
+	f := func() (interface{}, bool, error) {
+		expandedPool, err := GetStoragePoolByUUID(poolID)
+
+		if err != nil {
+			return nil, false, err
+		}
+		if expandedPool.LastOperation != nil {
+			if expandedPool.LastOperation.Status == opsapi.SdkStoragePool_OPERATION_FAILED {
+				return nil, false, fmt.Errorf("PoolResize has failed. Error: %s", expandedPool.LastOperation)
+			}
+		}
+		if expandedPool.LastOperation.Status == opsapi.SdkStoragePool_OPERATION_IN_PROGRESS {
+			// storage pool resize has been triggered
+			logrus.Infof("Pool %s expansion started", poolID)
+			return nil, true, nil
+		}
+		return nil, true, fmt.Errorf("pool %s resize not triggered ", poolID)
```

Comment:
If this method invoked immediately after pool expand there could be few chances that pool will not be on above two status and it is simply online.  This method will not wait right?

Reply:
yes, this logic is in rerty method, which checks for status to changes to in progress for a max of 2 mins in 10 second interval